wpimath/src/main/native/include/frc/geometry/Rotation3d.h

```diff
@@ -59,13 +59,21 @@ class WPILIB_DLLEXPORT Rotation3d {
    */
   Rotation3d(const Vectord<3>& axis, units::radian_t angle);
 
+  /**
+   * Constructs a Rotation3d with the given rotation vector representation. The
+   * axis doesn't have to be normalized.
```

Comment:
In fact, the axis _shouldn't_ be normalized, because that's an integral part of rotation vectors. Instead of mentioning normalization, I think it would be better to describe what a rotation vector is.

Reply:
Yes. This was sloppy on my part, I have this constructor documented in java and missed it when implementing the cpp side.